src/mqt/yaqs/core/methods/tdvp/integrators.py

```
@@ -0,0 +1,497 @@
+                "left",
+                dynamic=False,
+            )
+            right_blocks[i] = update_right_environment(
+                state.tensors[i + 1], state.tensors[i + 1], operator.tensors[i + 1], right_blocks[i + 1]
+            )
+
+        if drift_renorm and uses_fixed_chi(sim_params):
+            renorm_drift(state, sim_params)
+
+
+def sweep_dynamic(
```

Ruff (0.15.15)

[warning] Too many branches (18 > 12)

(PLR0912)